solidity/deploy/41_deploy_multi_asset_vault.ts

```diff
@@ -0,0 +1,46 @@
+import type { DeployFunction } from "hardhat-deploy/types"
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import { waitForTransaction } from "../helpers/deployment"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { deployments, helpers } = hre
+  const { deployer, governance } = await helpers.signers.getNamedSigners()
+
+  const mezoPortal = await deployments.get("MezoPortal")
+
+  const initialSupportedAssets = [
+    "0x7A56E1C57C7475CCf742a1832B028F0456652F97", // SolvBTC
+    "0xd9D920AA40f578ab794426F5C90F6C731D159DEf", // SolvBTC.BBN
+  ]
+
+  const [_, deployment] = await helpers.upgrades.deployProxy(
+    "AcreMultiAssetVault",
+    {
+      contractName: "AcreMultiAssetVault",
+      initializerArgs: [
+        await governance.getAddress(),
+        mezoPortal.address,
+        initialSupportedAssets,
+      ],
+      factoryOpts: { signer: deployer },
+      proxyOpts: {
+        kind: "transparent",
```

Comment:
Assuming the new behavior of the deploy plugin is to create a separate `ProxyAdmin` for each contract (to be confirmed, I am never sure which behavior is old and which one is new), we should add an `initialOwner` parameter to `proxyOpts`.

>  The proxy admin contract also defines an owner address which has the rights to operate it. By default, the proxy admin’s owner is the initialOwner address used during deployment of the transparent proxy if provided, otherwise it is the externally owned account used during deployment. 

https://docs.openzeppelin.com/upgrades-plugins/1.x/

Reply:
Amazing catch! 🎉 
293ea548bd269a77448c7340d0a62149e3e9227e